sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureAuthorizedClientRepository.java

```diff
@@ -64,7 +64,10 @@ public void saveAuthorizedClient(OAuth2AuthorizedClient client,
             OAuth2AuthorizedClient fakeAuthzClient = createFakeAuthzClient(azureClient, id, principal);
             OAuth2AuthorizationContext.Builder contextBuilder =
                 OAuth2AuthorizationContext.withAuthorizedClient(fakeAuthzClient);
-            String[] scopes = repo.findByRegistrationId(id).getScopes().toArray(new String[0]);
+            String[] scopes = null;
+            if (!AADWebAppClientRegistrationRepository.AZURE_CLIENT_REGISTRATION_ID.equals(id)) {
```

Comment:
Add method in AADWebAppClientRegistrationRepository:
```
public static final boolean isDefaultClient(String clientId) {...}
```

The method can be used in `AADWebAppConfiguration`, too.